src/dynamics/video_player/player/player.js

```diff
@@ -1328,7 +1329,7 @@ Scoped.define("module:VideoPlayer.Dynamics.Player", [
                                     if (/^[\d\s]+\*$/.test(item)) {
                                         item = +item.replace("\*", '');
                                         this.on("change:duration", function(duration) {
-                                            if (duration > 0 && this.get("midrollads").length === 0) {
+                                            if ((duration > 0) && this.get("midrollads").length === 0) {
```

Comment:
question (non-blocking): do the parenthesis fix the problem here?  Order of operations should be fine without it

Reply:
I forgot to remove that, I was testing something and add a conditional, and when removing it let behind the parenthesis...